app/views/users/line_Items.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Items</h1>
+<% @line_Item.each do |line_item| %>
```

Comment:
create a partial _line_item, and render it here

Reply:
done